logback-core/src/main/java/ch/qos/logback/core/BasicStatusManager.java

```diff
@@ -90,18 +97,18 @@ private void fireStatusAddEvent(Status status) {
 
     public void clear() {
         synchronized (statusListLock) {
-            count = 0;
+            count.reset();
             statusList.clear();
             tailBuffer.clear();
         }
     }
 
     public int getLevel() {
```

Comment:
Should `StatusManager#getLevel()` be uncommented? 

```java
package ch.qos.logback.core.status;
…
public interface StatusManager {
…
    /**
     * Return the highest level of all the statii.
     * 
     * @return
     */
    // int getLevel();
…
}
```

Should `@Override` be added to all implemented methods?